### DISCUSSION TITLE
Allowed languages for dictionary/spellcheck in Nextcloud Office (aka Collabora Online)?

### DISCUSSION THREAD

**Discussion by navigante** [nextcloud/all-in-one#1318] [Wiki]
It happened to me, that I didn`t define preferred Collabora dictionaries creating initial setup in AIO Panel. 
Only default languages seems to be available, but I was looking specially for Polish one.
 
Then I spent couple of hours wondering how to add pl_PL dictionary hunting for collabora container, coolwsd config file inside the container, proper environmental variables, researching collabora online forums, languagetool project and much more.

The solution is maybe not so obvious, but very simple. 

When you`ve got enabled Collabora add-on in AIO Panel, you have to stop containers to see the option for setting dictionaries again.
Be aware, that after container restart the information about dictionaries become invisible again.

What is my learning? 
The information about dictionaries chosen should be visible in anytime, in the same manner as information about time zones.

I hope it can save the time of others.

**Comment by szaimen** [comment]
Sorry, we are not making this visible like the timezone setting since it adds yet another option to the already crowded install wizard. You can simply change it after starting the container the first time.

However lets make a Wiki article out of this.
